source/api_cc/src/DeepPot.cc

```diff
@@ -78,6 +78,88 @@ run_model (ENERGYTYPE &			dener,
   atommap.backward (dforce_.begin(), dforce.begin(), 3);
 }
 
+#if HUAWEI_ASCEND
+static void 
+run_model (ENERGYTYPE &			dener,
```

Comment:
What is the difference between new and old functions?

Reply:
Thanks for the comment! We modify the function, so we have to add a new overloading function for ascend platform.  The modification is the followings.

1. we pad the input_tensors in order to make the model being a static computation graph, so the input_tensors is no longer a constant input.
2. Since we pad the input_tensors, the output "o_energy" becomes useless value. We compute "o_energy" value in post process.
3. We add an "Assign" op, in order to copy mesh data from host to device every N steps.